### DISCUSSION TITLE
Translated Transcription

### DISCUSSION THREAD

**Discussion by XiteXx** [chidiwilliams/buzz#632] [Q&A] [answered]
Hello wanted to know if this feature is already available on other platform or if im just not finding it on windows version.

Basically i wanted to use the app to tranlate live audio from my voice to a different language, for example i speak in english and the transcription in Spanish for example ?

**Comment by chidiwilliams** [comment]
> Hello wanted to know if this feature is already available on other platform or if im just not finding it on windows version.
> 
> Basically i wanted to use the app to tranlate live audio from my voice to a different language, for example i speak in english and the transcription in Spanish for example ?

Hi, Whisper only supports X-to-X transcription or X-to-English translation, not English-to-X.

**Comment by raivisdejus** [comment, marked as answer]
Latest versions of Buzz also support translations using AI. For usage see https://chidiwilliams.github.io/buzz/docs/usage/translations
